### ISSUE TITLE
Tries to run ghc-mod on binary files

### ISSUE THREAD

**Issue by fractalcat** [anchor/git-vogue#57]
```
ghc-mod check succeeded with:
GMEIOException /home/sio/src/borel/dist/setup-config: hGetContents: invalid argument (invalid byte sequence)
```

Probably shouldn't be running `ghc-mod check` on that file. (Or succeeding on an exception, but this is perhaps covered by #13?)

**Comment by christian-marie** [comment]
This is odd and not quite what it seems. It's almost certain that we are not pointing ghc mod at that file. We should be filtering by suffix. I'd be interested in how to reproduce this

**Comment by christian-marie** [comment]
Probably fixed in #63
